staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -194,19 +195,6 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		apiregistrationClient,
 		5*time.Minute, // this is effectively used as a refresh interval right now.  Might want to do something nicer later on.
 	)
-	kubeInformers := kubeinformers.NewSharedInformerFactory(c.CoreAPIServerClient, 5*time.Minute)
-
-	var routing ServiceResolver = c.ServiceResolver
-	if routing == nil {
-		if c.EnableAggregatorRouting {
```

Comment:
This value is no longer consumed anywhere here, correct? Please remove it from the config struct and fix the comments to say that the service resolver is mandatory.